src/test/pretty/block-comment-wchar.rs

```diff
@@ -3,7 +3,6 @@
 // Test via: rustc --pretty normal src/test/pretty/block-comment-wchar.rs
 // ignore-tidy-cr
 // ignore-tidy-tab
-// pp-exact:block-comment-wchar.pp
```

Comment:
Could you update `block-comment-wchar.pp` instead of removing it?

Reply:
Fixed!